qiskit/providers/fake_provider/fake_backend.py

```diff
@@ -399,6 +411,10 @@ def _get_noise_model_from_backend_v2(
                 "ignore",
                 module="qiskit.providers.aer.noise.device.models",
             )
+            warnings.filterwarnings(
```

Comment:
The filter on this warning above might be out-of-date now (albeit not directly related to this change).